cmd/up/up.go

```diff
@@ -63,36 +63,31 @@ type UpOptions struct {
 	// ManifestPathFlag is the option -f as introduced by the user when executing this command.
 	// This is stored at the configmap as filename to redeploy from the ui.
 	ManifestPathFlag string
-	// ManifestPath is the patah to the manifest used though the command execution.
+	// ManifestPath is the path to the manifest used though the command execution.
 	// This might change its value during execution
-	ManifestPath string
-	Namespace    string
-	K8sContext   string
-	DevName      string
-	Devs         []string
-	Envs         []string
-	Remote       int
-	Deploy       bool
-	ForcePull    bool
-	Reset        bool
+	ManifestPath     string
+	Namespace        string
+	K8sContext       string
+	DevName          string
+	Envs             []string
+	Remote           int
+	Deploy           bool
+	ForcePull        bool
+	Reset            bool
+	commandToExecute []string
```

Comment:
where are we consuming this? I mean, where is this command being executed?

Reply:
Those are the flags that the user can use when using the `okteto up` command

Reply:
Not really being executed but it is being passed into `dev.Command.Values` to be used later in [here](https://github.com/okteto/okteto/pull/3215/files#diff-c1b929a650f782cf212e9b4b0ced5a7a298598bcb7473be505e7af1f06d6cf30R278). Passing it just before `up.Dev = dev` makes sure that the commands (if any) in the [okteto manifest](https://www.okteto.com/docs/reference/manifest/#dev-object-optional) would be replaced by the commands entered using the cli. WDYT?

Reply:
Perfect! We should check if the dev command gets modified later, but sounds good :+1:

Reply:
:point_up: Checked. Almost all places are checking if empty to write a default value (from the manifest for example).. The only one that's not checking if empty is https://github.com/abs007/okteto/blob/e25e1a665d057662498aeeb10ba807ccbb4a163b/pkg/model/devrc.go#L90, which doesn't seem like an issue. @jLopezbarb can you confirm?

Reply:
It's not an issue. We expect to overwrite the command with the one used in the args